src/tools.ts

```diff
@@ -144,26 +144,14 @@ export const createKeyPair = () => {
   }
 }
 
-export interface JwtPayload {
-  sub?: string
-  iss?: string
-  aud?: string
-  exp?: string
-  nbf?: string
-  iat?: string
-  jti?: string
-}
-
 export const signJwt = (
   privateKey: forge.pki.PrivateKey,
   jwtPayload: JwtPayload,
   kid?: string
-) => {
-  const bufferedJwt = Buffer.from(JSON.stringify(jwtPayload))
-  return sign(bufferedJwt, forge.pki.privateKeyToPem(privateKey), {
+) =>
+  sign(jwtPayload, forge.pki.privateKeyToPem(privateKey), {
     header: { kid, alg: 'RS256' },
   })
```

Comment:
Now this function is much simpler. Maybe this also helps with #64 as a side effect.